dnf/dnssec/dnsseckeyverification.py

```diff
@@ -0,0 +1,237 @@
+import base64
+import hashlib
+import logging
+import re
+import unbound
+
+from enum import Enum
+from dnf.dnssec import DnssecError
+from dnf.i18n import _
+import dnf.rpm.transaction
+
+
+logger = logging.getLogger("dnf")
+
+# TODO: make this feature optional
+
+RR_TYPE_OPENPGPKEY = 61
+
+
+def email2location(email_address, tag="_openpgpkey"):
+    # type: (str, str) -> str
+    """
+    Implements RFC 7929, section 3
+    https://tools.ietf.org/html/rfc7929#section-3
+    :param email_address:
+    :param tag:
+    :return:
+    """
+    split = email_address.split("@")
+    if len(split) == 2:
```

Comment:
suggestion: 
if len(split) != 2:
   raise DnssecError()
and then save 4 spaces of indentation